libraries/ms-drivers/inc/mcp23008_gpio_expander_defs.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+// Internal MCP23008 register definitions and defines for accessing it
+// Datasheet: https://ww1.microchip.com/downloads/en/DeviceDoc/21919e.pdf
+
+#define I2C_PORT I2C_PORT_2
+
+#define CONFIG_PIN_I2C_SCL \
+  { GPIO_PORT_B, 10 }
+#define CONFIG_PIN_I2C_SDA \
+  { GPIO_PORT_B, 11 }
+
+#define IODIR 0x00    // direction, 1 input 0 output
+#define IPOL 0x01     // polarity, corresponding GPIO bit will show inverted value on the pin
+#define GPINTEN 0x02  // interrupt on change 1 enable 0 disable
+#define DEFVAL 0x03   // default compare value for interrupt, opposite value on pin causes interrupt
+#define INTCON 0x04   // interrupt control, interrupt-on-change vs interrupt-on-previous-val
+
+// IO configuration register
```

Comment:
remove these 5 lines of comments,

Reply:
I know I added them, but they don't really make sense when I read them now, which means they're useless lol